src/main/java/Main.java

```diff
@@ -0,0 +1,37 @@
+import dao.FruitDao;
+import dao.impl.FruitDaoImpl;
+import java.util.HashMap;
+import java.util.Map;
+import model.Transaction;
+import service.ParserService;
+import service.ReaderService;
+import service.WriterService;
+import service.impl.ParserServiceImpl;
+import service.impl.ReaderServiceImpl;
+import service.impl.ReportServiceImpl;
+import service.impl.WriteServiceImpl;
+import service.strategy.TransactionHandler;
+import service.strategy.impl.BalanceHandler;
+import service.strategy.impl.PurchaseHandler;
+import service.strategy.impl.ReturnHandler;
+import service.strategy.impl.SupplyHandler;
+
+public class Main {
+    private static final String INPUT_CSV_PATH = "src/main/resources/input.csv";
+    private static final String REPORT_CSV_PATH = "src/main/resources/report.csv";
+    private static final FruitDao FRUIT_DAO = new FruitDaoImpl();
+    private static final ReaderService READER = new ReaderServiceImpl();
+    private static final ParserService PARSER = new ParserServiceImpl();
+    private static final WriterService WRITER = new WriteServiceImpl();
```

Comment:
These 4 fields are not constants, better make them local and create them in the main(). 
FYI [constant naming](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)